### ISSUE TITLE
questions about learning rate

### ISSUE THREAD

**Issue by StephenZhao1** [meteorshowers/hed-pytorch#10]
this network has different layer with different learning rate, why was that?
